src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
+// const deer = new Herbivore('Bembi');
+// const panther = new Carnivore('Bagira');
+// const lion = new Carnivore('King');
+// const rabbit = new Herbivore('Max');
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  name = this.name;
+  health = this.health;
+  hidden = false;
```

Comment:
There's no need to re-declare the `name` and `health` properties in the `Herbivore` class since they are already declared in the `Animal` class and inherited by `Herbivore`. This is a code quality issue as it introduces unnecessary code duplication.